client/util.h

```diff
@@ -149,7 +149,7 @@ extern void AddLogLine(char *fileName, char *extData, char *c);
 extern void AddLogHex(char *fileName, char *extData, const uint8_t * data, const size_t len);
 extern void AddLogUint64(char *fileName, char *extData, const uint64_t data);
 extern void AddLogCurrentDT(char *fileName);
-extern void FillFileNameByUID(char *fileName, uint8_t * uid, char *ext, int byteCount);
+extern void FillFileNameByUID(char *fileName, uint8_t * uid, const char *ext, int uidlen);
```

Comment:
if you are change parameter names,   do be consistant with the change

Reply:
Fixed, thanks